integration/ingester_test.go

```diff
@@ -501,9 +501,8 @@ func TestIngesterQuerying(t *testing.T) {
 					defer s.Close()
```

Comment:
Given streaming chunks from ingesters is now always enabled, we can remove the `for _, streamingEnabled := range []bool{true, false} {` above.

Reply:
Done